src/devices/Seesaw/SeesawEncoder.cs

```diff
@@ -0,0 +1,92 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Buffers.Binary;
+
+namespace Iot.Device.Seesaw
+{
+    public partial class Seesaw : IDisposable
+    {
+        /// <summary>
+        /// Read the current position of the encoder
+        /// </summary>
+        /// <param name="encoder">Which encoder to use, defaults to 0</param>
+        /// <returns>The encoder position as a 32 bit signed integer</returns>
+        /// <exception cref="InvalidOperationException"></exception>
```

Comment:
Would be nice to put a description here. Also, for all comments, best to finish with a dot.